### PR TITLE
Fix note on `model.optimizer.learning_rate`

### PR DESCRIPTION
**Pull request by darthdeus**
The `model.optimizer.learning_rate` property already returns the LR tensor directly. See https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/optimizers/Optimizer#hyper_parameters
as an example.

### CODE REVIEW

**Comment by foxik** [comment]
Not really. The `model.optimizer.learning_rate` generally returns whatever was used during initialization, and if you pass a `LearningRateSchedule` (which is the condition at the beginning of the sentence you are correcting), then you get the `LearningRateSchedule` back. This class has a `__call__` method, which given the current step number returns the learning rate to use. All Keras V2 optimizers know this class and handle it transparently. The documentation is still lacking, but https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/optimizers/schedules/ExponentialDecay contains an example of passing the schedule directly as a learning rate.

(I am actually using the code in my solution, so it definitely works, at least with TF 2.0.0a0.)

**Comment by darthdeus** [comment]
> The model.optimizer.learning_rate generally returns whatever was used during initialization

That explains it. Though then the comment above is a bit misleading in combination with this

> If `args.decay` is not specified, pass the given `args.learning_rate` directly to the optimizer

because if `args.learning_rate` is passed through then it won't work. I'm not sure if there's a good solution, since there is no constant schedule. I guess it's either check if it's a tensor, or pass a schedule with _constant_ schedule with the same start/end, neither of which seem nice. Probably does not happen in normal code which would use schedules anyway.

**Comment by foxik** [comment]
I am not sure I understand -- you mean that `model.optimizer.learning_rate(model.optimizer.iterations)` does not work when there is no decay? That is so, but I do not care (and the comment in the source specifies that the `lr(iters)` only works if there is a decay schedule set).

The easiest way to get a constant schedule would be `tf.keras.optimizers.schedules.PiecewiseConstantDecay([0], [lr, lr])` -- unfortunately, the boundaries cannot be empty, `PiecewiseConstantDecay([], [lr])` would be quite intuitive.
